usr/share/rear/prep/GNU/Linux/220_include_lvm_tools.sh

```diff
@@ -13,3 +13,9 @@ fsadm
 COPY_AS_IS=( "${COPY_AS_IS[@]}"
 /etc/lvm
 )
+
+if lvs --noheadings -o thin_count | grep -q -v "^\s*$" ; then
+    # There are Thin Pools on the system, include required binaries
+    PROGS=( "${PROGS[@]}" /usr/sbin/thin_* )
```

Comment:
@rmetrich Could you list up (in comment) which executables you need at minimum? I think of thin_check (aka RedHat Solution bulletin 3031921 - https://access.redhat.com/solutions/3031921)
Why do I ask this? I know it has been tested and approved for RHEL7 by @schabrolles - but what about SLES? Are the executables named the same? To be verified by @jsmeix 

Personally I will approve once @jsmeix gives his blessing.

Reply:
I cannot test or verify this within a reasonable time,
not this week and next week I am not in the office
and afterwards - with probability one - some other
issues have piled up that have higher priority.

Reply:
In fact they are not really needed except `thin_check`, since for thin volumes, we recreate these using `lvcreate` commands, so I'll remove the others (only the libraries are needed).
Let me test again without ...